action.yml

```diff
@@ -28,6 +36,6 @@ inputs:
   postCommands:
     description: "Commands to execute after publishing the Workers project"
     required: false
-  publish:
-    description: "Set to false to skip publishing your Worker project and secrets.  Defaults to true."
-    required: false
\ No newline at end of file
+  command:
+    description: "The Wrangler command you wish to run. For example: publish - this will publish your Worker"
+    required: true
```

Comment:
I think we probably want the default `command` to be `publish`, just so it's as little friction for people upgrading from v1 as possible. I think wrangler2 will still grab the entrypoint from the various places it can be specified (although you might get warnings to suggest you instead make it explicit in the command).

Reply:
Agree, I would be for having `command` optional, and you are already setting it as default when the input is not present :)
https://github.com/cloudflare/wrangler-action/pull/72/files#diff-6f9d41d046756f0ddc2fcee0626bdb50100d12b88f293734eff742818e03efa2R126